ci/generate-html.sh

```diff
@@ -5,4 +5,8 @@ npm install
 # copy data generated in previous step into place
 cp ../data/data.json src
 
-npm run generate-html
\ No newline at end of file
+# compile CSS
+npx gulp compile
```

Comment:
Is this meant to be npx?

Reply:
Yup, see https://designsystem.digital.gov/documentation/getting-started/developers/phase-two-compile/#step-6-export-compile-functions-2

`npx` is a shortcut for running a command on a locally installed node module. so `npx gulp compile` means run `compile` on the locally installed `gulp` module

Reply:
https://docs.npmjs.com/cli/v9/commands/npx/

Reply:
Oh cool, not something I was familiar with. Looks good!